### PR TITLE
feat: search in more than one namespace

### LINKED ISSUE #18: Allow to search in more than one namespace
Something like:

```bash
kgrep logs -n namespace1,namespace2 -r myapp -p my_search
```

### PR DESCRIPTION
**Pull request by matheusandre1**
Closes: #18

### CODE REVIEW

File: cmd/logs.go [hunk 1]
```diff
@@ -60,7 +75,9 @@ var logsCmd = &cobra.Command{
 			}
 
 			if err != nil {
-				return fmt.Errorf("failed to search logs in namespace %q: %v", ns, err)
+				if ns != "" {
+					return fmt.Errorf("failed to search logs in namespace %q: %v", ns, err)
+				}
```
Comment (hbelmiro) [inline cmd/logs.go:91]: Don't swallow the error when `ns` is nil.

```suggestion
				if ns != "" {
					return fmt.Errorf("failed to search logs in namespace %q: %v", ns, err)
				}
				return fmt.Errorf("failed to search logs: %v", err)
```

File: cmd/logs.go [hunk 2]
```diff
@@ -36,7 +36,22 @@ var logsCmd = &cobra.Command{
...
+				if _, ok := seen[t]; ok {
+					continue
+				}
+				seen[t] = struct{}{}
+				namespaces = append(namespaces, t)
+			}
+			if len(namespaces) == 0 {
+				return fmt.Errorf("invalid namespace list: no valid namespaces provided")
+			}
```
Comment (hbelmiro) [inline cmd/logs.go]: The namespace parsing and dedup logic (lines 38-54) has no dedicated unit test. `TestLogsCommand_MultiNamespace` only validates that the CLI flag syntax is accepted; it cannot exercise the actual parsing because `NewLogGrepper()` fails without a kubeconfig. Edge cases such as trailing commas (`"ns1,"`), all-whitespace tokens (`"ns1, ,ns2"`), and duplicate namespaces (`"ns1,ns1"`) are handled in code but untested.

**Suggested fix:** extract the namespace-parsing block into a standalone function (e.g. `parseNamespaces(raw string) ([]string, error)`) and add table-driven tests covering empty input, single namespace, duplicates, whitespace-only tokens, and trailing commas.

File: cmd/logs.go
Comment (hbelmiro) [inline cmd/logs.go]: `ns = strings.TrimSpace(ns)` is redundant. Every token appended to `namespaces` was already trimmed at line 42 (`t := strings.TrimSpace(tok)`), and the empty-namespace fallback at line 56 (`namespaces = []string{""}`) cannot benefit from trimming either. The dead code adds a false impression that untrimmed values could reach this point.

**Suggested fix:** remove line 61.
